src/coreclr/pal/src/arch/riscv64/context2.S

```diff
@@ -9,4 +9,246 @@
 #include "unixasmmacros.inc"
 #include "asmconstants.h"
 
-#error "TODO-RISCV64: missing implementation"
+// Incoming:
+//  a0: Context*
+//  a1: Exception*
+//
+LEAF_ENTRY RtlRestoreContext, _TEXT
+#ifdef HAS_ASAN
+    lw  t1, CONTEXT_ContextFlags(a0)
+    andi t1, t1, 0x1 << CONTEXT_FLOATING_POINT_BIT
```

Comment:
In arm64 implementation in a similar place CONTEXT_CONTROL_BIT is used instead of CONTEXT_FLOATING_POINT_BIT. Isn't this a bug in loongarch64 implementation?

Reply:
This time I think we don't need ASAN support for RISC-V.
I removed those and left a TODO comment.